cmd/libsnap-confine-private/cgroup-support.h

```diff
@@ -37,4 +37,28 @@ void sc_cgroup_create_and_join(const char *parent, const char *name, pid_t pid);
  **/
 bool sc_cgroup_is_v2(void);
 
+/**
+ * sc_cgroup_is_tracking_snap checks whether the snap process are being
```

Comment:
I suppose this needs clarification, what this is doing is checking if there are processes belonging to the snap that are not this process already being tracked is that right?

s/the snap process are/any snap process other than the caller is/ ?

maybe

Reply:
updated